data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -350,12 +350,14 @@ then
 			--dest-dir=/assets/cco-bootstrap \
 			--manifests-dir=/assets/manifests \
 			--cloud-credential-operator-image=${CLOUD_CREDENTIAL_OPERATOR_IMAGE}
+			{{- if .CloudCredentialsMode}} \
+			--force-mode={{.CloudCredentialsMode}}
+			{{- end}}
```

Comment:
I am not in favor of adding template based decision making in bootkube.sh

the cred-minter has access to install-config.yaml as config-map on disk, I would recommend that cred-minter use that to decide instead of templates..

Reply:
Yep, that makes sense. Thanks.